src/current/v24.2/logging-best-practices.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Logging Best Practices
+summary: Best practices for consuming CockroachDB logs for critical information.
+toc: true
+---
+
+This page provides guidance and best practices for consuming CockroachDB logs for critical information. It outlines how to externalize logs. Refer to [Critical Log Messages]({% link {{ page.version.version }}/critical-log-messages.md %}) for which log messages to externalize, and how to interpret them.
+
+## Use network log sinks
+
+Output different [logging channels]({% link {{ page.version.version }}/logging.md %}#logging-channels) over the network to third-party log aggregators:
+
+ - [Output to Fluentd-compatible network collectors]({% link {{ page.version.version }}/configure-logs.md %}#output-to-fluentd-compatible-network-collectors), such as Elasticsearch, Splunk.
+ - [Output to HTTP network collectors]({% link {{ page.version.version }}/configure-logs.md %}#output-to-http-network-collectors), such as Datadog. 
+
+## Use `json` format
```

Comment:
I recall json log format being problematic in some Support situations - was that a consideration here, are we concerned about that moving forward?

Reply:
discussed with @kevin-v-ngo.  I will clarify: "With third-party tool consumption".

Reply:
updated